backend/app/forms/guest_submission_form.rb

```diff
@@ -0,0 +1,73 @@
+class GuestSubmissionForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  # Guest の属性
+  attr_accessor :first_name, :middle_name, :last_name, :guest_side
+  
+  # GuestPersonalInfo の属性
+  attr_accessor :email, :postal_code, :prefecture_code, :city_code, :town, :building
+
+  # GuestAnswer の属性
+  attr_accessor :attendance, :allergy, :message
+
+  # モデルインスタンス
+  attr_reader :guest, :guest_personal_info, :guest_answer
+
+  # バリデーション
+  # Guest
+  validates :first_name, presence: true, length: { maximum: 100 }
+  validates :middle_name, length: { minimum: 1, maximum: 100 }, allow_blank: true
+  validates :last_name, presence: true, length: { maximum: 100 }
+  validates :guest_side, presence: true, numericality: { only_integer: true, less_than_or_equal_to: 1 }
+  # GuestPersonalInfo
+  validates :email, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP}, length: { maximum: 255 }
+  validates :postal_code, presence: true, length: { maximum: 8 }
+  validates :prefecture_code, presence: true, numericality: { only_integer: true, less_than_or_equal_to: 99 }
+  validates :city_code, presence: true, numericality: { only_integer: true, less_than_or_equal_to: 99999 }
+  validates :town, presence: true, length: { maximum: 100 }
+  validates :building, length: { maximum: 100 }
+  # GuestAnswer
+  validates :attendance, presence: true, numericality: { only_integer: true, less_than_or_equal_to: 1 }

```

Comment:
```suggestion

  # Guest
  validates :first_name, presence: true, length: { maximum: 100 }
  validates :middle_name, length: { minimum: 1, maximum: 100 }, allow_blank: true
  validates :last_name, presence: true, length: { maximum: 100 }
  validates :guest_side, presence: true, numericality: { only_integer: true, less_than_or_equal_to: 1 }

  # GuestPersonalInfo
  validates :email, presence: true, format: { with: URI::MailTo::EMAIL_REGEXP}, length: { maximum: 255 }
  validates :postal_code, presence: true, length: { maximum: 8 }
  validates :prefecture_code, presence: true, numericality: { only_integer: true, less_than_or_equal_to: 99 }
  validates :city_code, presence: true, numericality: { only_integer: true, less_than_or_equal_to: 99999 }
  validates :town, presence: true, length: { maximum: 100 }
  validates :building, length: { maximum: 100 }

  # GuestAnswer
  validates :attendance, presence: true, numericality: { only_integer: true, less_than_or_equal_to: 1 }
```

- 自明なコメントは不要
- 改行で意味を持ったグループに分ける